crates/astria-conductor/src/config.rs

```diff
@@ -66,10 +66,14 @@ where
 }
 
 impl Config {
-    fn from_environment() -> Result<Config, figment::Error> {
+    fn from_environment(prefix: Option<&str>) -> Result<Config, figment::Error> {
```

Comment:
it appears `None` is the only value passed in here (excluding tests), maybe just remove the param and the `Some` case?